addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -19,6 +19,10 @@ var DebouncedField = basicFields.DebouncedField;
 var JournalDashboardGraph = basicFields.JournalDashboardGraph;
 var _t = core._t;
 
```

Comment:
not needed ? as we have a fall back in basic_fields (odoo.session_info.max_file_upload_size || 128 * 1024 * 1024);

Reply:
`session_info` is undefined in the test